MobileBuy/sample/src/main/java/com/shopify/sample/customer/CustomerOrderListViewPresenter.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *   The MIT License (MIT)
+ *
+ *   Copyright (c) 2015 Shopify Inc.
+ *
+ *   Permission is hereby granted, free of charge, to any person obtaining a copy
+ *   of this software and associated documentation files (the "Software"), to deal
+ *   in the Software without restriction, including without limitation the rights
+ *   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *   copies of the Software, and to permit persons to whom the Software is
+ *   furnished to do so, subject to the following conditions:
+ *
+ *   The above copyright notice and this permission notice shall be included in
+ *   all copies or substantial portions of the Software.
+ *
+ *   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *   THE SOFTWARE.
+ */
+package com.shopify.sample.customer;
+
+import com.shopify.buy.dataprovider.Callback;
+import com.shopify.buy.dataprovider.CancellableTask;
+import com.shopify.buy.dataprovider.RetrofitError;
+import com.shopify.buy.model.Customer;
+import com.shopify.buy.model.Order;
+import com.shopify.sample.WeakObserver;
+import com.shopify.sample.BaseViewPresenter;
+import com.shopify.sample.application.SampleApplication;
+
+import java.lang.ref.WeakReference;
+import java.util.List;
+
+import rx.Subscription;
+import rx.functions.Action2;
+
+public final class CustomerOrderListViewPresenter extends BaseViewPresenter<CustomerOrderListViewPresenter.View> {
+
+    public static interface View extends BaseViewPresenter.View {
+
+        void showOrderList(List<Order> orders);
+
+    }
+
+    private CancellableTask fetchOrderListTask;
+
+    @Override
+    public void attach(final View view) {
+        super.attach(view);
+
+        final Customer customer = SampleApplication.getCustomer();
+        if (customer != null) {
+            // fetchCustomerOrdersWithRx(customer);
```

Comment:
How about a variable in SampleApplication that switches RxJava on/off instead of commenting this out?
```
if (SampleApplication.useRx) {
  fetchCustomerOrdersWithRx(customer)
} else {
...
}